README.md

```diff
@@ -1,24 +1,26 @@
-# README
+# Unnamed-Rails-Project
```

Comment:
this file is not a part of travis feature

Reply:
You said that readme part with build status should be a part of Travis pr...